tools/src/main/java/org/apache/kafka/tools/StreamsResetter.java

```diff
@@ -112,6 +115,7 @@ public class StreamsResetter {
             + "reset tool!\n\n"
             + "*** Warning! This tool makes irreversible changes to your application. It is strongly recommended that "
             + "you run this once with \"--dry-run\" to preview your changes before making them.\n\n";
+    private static final Logger log = LoggerFactory.getLogger(StreamsResetter.class);
```

Comment:
Cf my comment below. We should not use an actual `Logger` -- let's remove this again.